### ISSUE TITLE
Sorting by Newest first / Oldest first is broken

### ISSUE THREAD

**Issue by iafan** [translate/pootle#4992]
1. Go to any browser page, e.g. https://translate.evernote.com/ru/mac_v5_evernote/translate/#filter=incomplete
2. Try choosing `Newest first` or `Oldest first` sorting method.

Expected: the units are sorted according to the UI setting.

Observed: changing the setting does nothing.

**Comment by iafan** [comment]
Julen also suggested that this bug may be somehow connected with #4987

**Comment by iafan** [comment]
Thanks for fixing this so quickly, @phlax!

**Comment by iafan** [comment]
We pushed the release today with this fix being a part of the changes. I was testing it on https://translate.evernote.com/ru/support_evernote/translate/#filter=incomplete and selecting seort order still does nothing: though the URL hash is now updated with the sort parameter, choosing another sorting method actually does nothing on the data.

Note that if I select all units (no `#filter=incomplete`) then sorting method seems to have some effect.
